site/source/_posts/2018/03/11/Dojo-2-rc1.md

```diff
@@ -0,0 +1,186 @@
+
+As such, if a viable foundational approach exists that we can leverage or extend beyond the core of Dojo 2, we chose that route rather than creating our own. By leveraging tools like webpack for building and optimizations, various shims for custom elements and Intersection Observers, Globalize and cldr.js for internationalization, and more. On top of this foundation, we've built a reliable and consistent out of the box approach that makes it easy to get started building applications. Dojo 2 does not require each developer to make every architectural choice up front, but still provides easy ways to override our suggested approach when you prefer an alternative.
+
+There are already plenty of [articles that compare frameworks](https://www.sitepen.com/blog/2017/06/13/if-we-chose-our-javascript-framework-like-we-chose-our-music/), but needless to say, we took a long look at everything else because we want Dojo to be part of the ecosystem rather than its silo. By leveraging ES modules, TypeScript interfaces, and custom elements, we provide a robust level of interoperability with other frameworks and libraries.
+
+Our focus on developer ergonomics is to get thousands of small details to feel just right. For example, instead of manually worrying about code splitting, we handle this optimization out of the box with zero configuration. Or when you want to style a component, the list of class names available to that component is type safe, saving time in looking up the relevant class names from your CSS file.
+
+So what are Dojo's primary focus and advantages? It's an ergonomic way to build reactive enterprise web apps with TypeScript and numerous modern web standards and best practices, with an easy to use widget system and widget library, and other vital architectural components including routing, data stores, and much more.
+
+## Author widgets with `@dojo/widget-core`
+
+Dojo has always emphasized the creation of widgets, and Dojo 2's [`@dojo/widget-core`](http://github.com/dojo/widget-core) package creates an environment that provides developers with simple ergonomics to allow them to build robust applications. Leveraging a modern and reactive, uni-directional approach to data binding with a virtual DOM, `@dojo/widget-core` streamlines the process of creating widgets and apps.
+
+There are many scenarios where it is challenging to preserve a reactive architecture when working with properties of DOM nodes, so `@dojo/widget-core` provides meta abstractions to help deal with DOM nodes without breaking the flexibility of a reactive architecture.
+
+`@dojo/widget-core` embraces the future of the web platform with out of the box meta implementations for Intersection Observers, Web Animations, and other APIs, with plans shortly for additional emerging standards such as ResizeObserver.
+
+An example of a Dojo 2 widget using the Intersection Observer meta is:
```

Comment:
"An example of a Dojo 2 widget using the Intersection Observer meta is:" -> "An example that creates a list of images, but only requests the images when the become visible using the `Intersection` meta."

Reply:
Also do we think code blocks or images are nicer for these?

Reply:
In the README on widget-core we use this description, so some kind of mix of the two might be best.

"This example renders a list with images, the image src is only added when the item is in the viewport which prevents needlessly downloading images until the user scrolls to them:"